CMakeLists.txt

```diff
@@ -325,6 +325,9 @@ endif(WITH_XIO)
 
 #option for RGW
 option(WITH_RADOSGW "Rados Gateway is enabled" ON)
+if(WITH_RADOSGW)
```

Comment:
On 22-8-2016 14:22, Kefu Chai wrote:
> In CMakeLists.txt
> <https://github.com/ceph/ceph/pull/10517#discussion_r75666926>:
> 
>> @@ -325,6 +325,9 @@ endif(WITH_XIO)
>>  
>>  #option for RGW
>>  option(WITH_RADOSGW "Rados Gateway is enabled" ON)
>> +if(WITH_RADOSGW)
> 
> could you drop this commit? this message does not help our user, imho.

I know, this is going to be dropped.
But one way or another the Jenkins stuff refuses to add_directory(rgw).
Which it shows in the Jenkins output, it complains about missing
radosgw-admin.

Trying to figure out why in my CentOS build, but that all of a sudden
REQUIRED python3. So needed to fix that first.
And it is rather busy at the dayjob.